pint_xarray/tests/test_conversion.py

```diff
@@ -358,6 +358,8 @@ def test_convert_units(self, type, variant, units, error, match):
         )
 
         if error is not None:
+            if type == "DataArray":
+                obj = obj["a"]
```

Comment:
could you move this up two lines and remove the assignment below?

Reply:
sure